docs/source/contribute/dev-assisted-package-creation.rst

```diff
@@ -0,0 +1,113 @@
+Developer Assisted Package Creation
+===================================
+
+If you are a user, developer, or researcher that has software related to geospatial data
+processing and are unable to integrate that functionality into GeoIPS, you've come to
+the right place. Some of our developers for GeoIPS (specifically those that are CIRA-based), might be
+able to help you integrate your functionality into GeoIPS. Please continue reading if
+you think this may be of use!
+
+Why We Assist With Package Creation
+-----------------------------------
+
+The GeoIPS Development team is committed to integrating a variety of geospatial
+processing routines into GeoIPS, or as standalone packages that can interact with
+GeoIPS. This can be as simple as adding your algorithm or reader to the main GeoIPS
+code, or it can be more complex in creating a completely separate GeoIPS plugin-package
+specific for your use case.
+
+Please feel free to view any of our
+`public repositories <https://github.com/orgs/NRLMMD-GEOIPS/repositories>`_ for a
+general idea of what we can help you with. For any standalone package that we'll be
+developing for users who have requested assistance, we'll largely be following the
+structure of the `template_basic_plugin package <https://github.com/NRLMMD-GEOIPS/template_basic_plugin>`_.
+The aforementioned package is a template for creating separate plugin packages that can
+interact with GeoIPS seamlessly.
+
+The purpose of integrating software like this is to elevate GeoIPS as a package that can
+process nearly any type of satellite, in-situ, geo-located -derived data, and output it
+in a variety of formats. For each standalone package that we create, we only elevate the
+capabilities of GeoIPS as a whole. Eventually, we'd like to see GeoIPS be one, if not
+the package you need to handle your geospatial processing routines.
+
+Funding Requirements for Complex Transitions
+--------------------------------------------
+
+While we welcome all conversations about integrating external packages into GeoIPS, please note that if transitioning
```

Comment:
Suggestion:

"We are able to provide limited support for smaller integration tasks and welcome all conversations about integrating external packages into GeoIPS. Please note, however, that we are 100% soft-money funded. So, if transitioning your package requires substantial development time and resources, we may require funding to support that effort."

Reply:
Adding! Thanks